pillarbox-demo/src/main/java/ch/srgssr/pillarbox/demo/ui/player/controls/PlayerTimeSlider.kt

```diff
@@ -73,18 +80,25 @@ fun PlayerTimeSlider(
     progressTracker: ProgressTrackerState = rememberProgressTrackerState(player = player, smoothTracker = true),
     interactionSource: MutableInteractionSource = remember { MutableInteractionSource() },
 ) {
-    val duration by player.durationAsState()
+    val coroutineScope = rememberCoroutineScope()
+    val durationFlow = remember(player) {
+        player.durationAsFlow().map { durationMs ->
+            if (durationMs == C.TIME_UNSET) Duration.ZERO
+            else durationMs.milliseconds
+        }.stateIn(coroutineScope, SharingStarted.Lazily, ZERO)
+    }
+    val duration by durationFlow.collectAsState()
```

Comment:
Why not use the existing `Player.durationAsState()`?
```suggestion
    val durationMs by player.durationAsState()
    val duration = remember(durationMs) {
        if (durationMs == C.TIME_UNSET) ZERO
        else durationMs.milliseconds
    }
```